app/views/layouts/assignment_content.html.erb

```diff
@@ -2,7 +2,7 @@
 <html lang="<%= I18n.locale %>">
 <head>
   <%= render partial: 'layouts/head' %>
-  <title><%= t(:markus) %> - <%= controller.action_name.titleize %></title>
+  <title><%= content_for?(:html_title) ? yield(:html_title) : t(:markus) + ' - ' + controller.action_name.titleize%></title>
```

Comment:
`t(:markus)` should always appear. But you can use the `yield` to replace the controller action name :+1:

Reply:
Also, there should always be a space inside `<%=` and `%>` (fix throughout)